caller.hpp

```diff
@@ -0,0 +1,199 @@
+#ifndef CALLER_H
+#define CALLER_H
+
+#include <iostream>
+#include <algorithm>
+#include <functional>
+#include <cmath>
+#include <limits>
+#include "vg.pb.h"
+#include "vg.hpp"
+#include "hash_map.hpp"
+#include "utility.hpp"
+#include "pileup.hpp"
+
+namespace vg {
+
+using namespace std;
+
+// Super simple variant caller, for now written to get bakeoff evaluation bootstrapped.
+// Idea: Idependently process Pileup records, using simple thresholding to make calls.
```

Comment:
Oops, yeah, I added support for quality scores but didn't update that
comment (maybe others?).  Not sure if worth another pull request but happy
to do it..

On Wed, Sep 23, 2015 at 9:55 AM, Erik Garrison <notifications@github.com>
wrote:

> In caller.hpp <https://github.com/ekg/vg/pull/105#discussion_r40205281>:
>
> > +#include <algorithm>
> > +#include <functional>
> > +#include <cmath>
> > +#include <limits>
> > +#include "vg.pb.h"
> > +#include "vg.hpp"
> > +#include "hash_map.hpp"
> > +#include "utility.hpp"
> > +#include "pileup.hpp"
> > +
> > +namespace vg {
> > +
> > +using namespace std;
> > +
> > +// Super simple variant caller, for now written to get bakeoff evaluation bootstrapped.
> > +// Idea: Idependently process Pileup records, using simple thresholding to make calls.
>
> You're no longer just thresholding though, no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ekg/vg/pull/105/files#r40205281>.
>